src/keri/app/keeping.py

```diff
@@ -251,6 +258,17 @@ def __init__(self, headDirPath=None, perm=None, reopen=False, **kwa):
         if perm is None:
             perm = self.Perm  # defaults to restricted permissions for non temp
 
+        # support separate Keeper size config yet fall back to baser size if not set
+        keeperSize = os.getenv(KERIKeeperMapSizeKey, None)
+        baserSize = os.getenv(KERIBaserMapSizeKey, None)
+        mapSize = keeperSize if (keeperSize is not None and keeperSize != '') else baserSize
```

Comment:
You can reduce the size of this change with:

```
mapSize = os.getenv(KERIKeeperMapSizeKey) or os.getenv(KERIBaserMapSizeKey)
if mapSize:
    try:
        self.MapSize = int(mapSize)
    except ValueError:
        logger.error("KERI_KEEPER_MAP_SIZE and KERI_BASER_MAP_SIZE must be an integer value >1!")
        raise
```

`or` will provide the fallback you're adding; if the first expression is falsey, it evaluates the second expression.
`getenv` returns None by default.

```
def getenv(key, default=None):
    """Get an environment variable, return None if it doesn't exist.
    The optional second argument can specify an alternate default.
    key, default and the result are str."""
    return environ.get(key, default)
    ```

Reply:
I will change them to that. I'll also add an INFO log for the map size set.